tls-client/main.cpp

```diff
@@ -401,12 +412,11 @@ class HelloHTTPS {
     void onError(TCPSocket *s, int error) {
```

Comment:
This function has only a single call on line 210 and the string it prints conveys no additional information. I think the call on line 210 can be replaced with `_tcpsocket->close();` and this function can be deleted.